lib/generator.rb

```diff
@@ -35,13 +37,43 @@ def test_cases
     cases.call(data)
   end
 
+  def metadata_repository_missing_message
+    <<-EOM.gsub(/^ {6}/, '')
+
+      '#{METADATA_REPOSITORY}' repository not found.
+      Try running the command:
+        git clone https://github.com/exercism/#{METADATA_REPOSITORY}.git "#{metadata_dir}"
```

Comment:
Good suggestion, but I prefer keeping the message short. Anyone who has a ssh key set up should know how to change the repository URL.